### PR TITLE
update bias init&&update obj loss

### PR DESCRIPTION
**Pull request by Laughing-q**
improve accuracy without additional computation

### CODE REVIEW

**Comment by Laughing-q** [comment]
@AyushExel Actually the update in `yolo.py` is important, which make the huge improvement, you can see the results in this [link](https://wandb.ai/laughing/runs/runs/3tf1io0m?workspace=user-laughing). And the loss I update in `seg_loss.py` still got a little better accuracy, so I keep the update of the loss.

**Comment by Laughing-q** [comment]
@AyushExel Now I'll train a new benchmark of yolov5s-seg, what do you think?

**Comment by AyushExel** [comment]
@Laughing-q yes please! btw what hyperparameters will you use? The ones from hyp-scratch-low.hyp or modified?

**Comment by Laughing-q** [comment]
@AyushExel the current hyp seems to be suitable for yolov5s-seg, so I am using hyp-scratch-low.yaml.

**Comment by AyushExel** [comment]
@Laughing-q oh btw, the batch-size issue is also fixed by this PR or it still needs to be tweaked? I'm also going to run benchmark for m model and I was thinking of using only 2 GPUs with large batch size as they are 80gb cards

**Comment by Laughing-q** [comment]
@AyushExel When I was looking for the bs issue, I had done experiments with this update and it seemed the bs issue still needs to be tweaked. but I'm not so sure about this, maybe you can do a experiment to confirm.
